### ISSUE TITLE
False positive using ImportOrderCheck rule 

### ISSUE THREAD

**Issue by softdays** [checkstyle/sonar-checkstyle#20]
It seems that the ordered flag (which is default to true) doesn't work properly with the rule checkstyle:com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck
For example, if you try configure your Eclipse Neon to automatically organize your imports, you will have something like that:
```java
package my.app;

import java.io.Serializable;

import my.app.Service;
```
The third line will throw an alert: 
`Wrong order for 'fr.edu.grenoble.composant.securite.ComposantSecurite' import`

Same error with groups containing several imports which are nevertheless properly sorted alphabetically by Eclipse.

**Comment by romani** [comment]
please report issues on checkstyle project - https://github.com/checkstyle/checkstyle/issues
Sonar plug do nothing addition except for integration of validation to Sonar system
